allennlp/service/server_sanic.py

```diff
@@ -9,22 +9,23 @@
 LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
 LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
 
-# TODO(joelgrus): make this configurable
-servables = ServableCollection.default()  # pylint: disable=invalid-name
 
 app = Sanic(__name__)  # pylint: disable=invalid-name
```

Comment:
Is there a way to avoid constructing this at the module scope?  If we only construct this app when we need it, instead of on import, I'm pretty sure that would mean we could remove the logging hacks on line 9 and 10, and then unpin the sanic version.